ui/core/components/inputs/consumables.ts

```diff
@@ -887,13 +900,23 @@ export const MinorManaOil = (slot: ItemSlot): ConsumableInputConfig<WeaponImbue>
 };
 
 // Sharpening Stones
+export const ConsecratedSharpeningStone = (slot: ItemSlot): ConsumableInputConfig<WeaponImbue> => {
```

Comment:
This and blessed oil aren't unlocked until Naxx comes out. Are these/is there a way to hide this behind some phase boundary? Or rather, probably ask Lucenia how she wants to handle visibility for these.

Reply:
I think these are okay to leave available in the UI. There are other things that are from later phases (ZG world buff, smoked desert dumplings) and as long as they aren't enabled by default in the presets it should be okay.